### DISCUSSION TITLE
Slash commands in DMs

### DISCUSSION THREAD

**Discussion by sdanialraza** [discordjs/discord.js#9126] [Q&A] [answered]
Why are slash commands enabled in DMs by default when bots are mostly used in guilds, and commands are made while keeping guilds in mind?

**Comment by RobbiDev** [comment]
I'm not the creator of this feature and am just speculating. But I assume it's enabled by default for two reasons. One reason is that you can simply DM the bot instead of searching for the "bot-command" channel, which is full of spam from other bots. The second reason is most likely in larger servers. sending a slash command with multiple users using it at the same time. Can cause the bot's response to get lost within other users' responses. Trying to make DMs a more practical way instead of a guild channel.

**Reply by sdanialraza** [reply]
It's a great option to have for commands, but why have it enabled by default when most users on Discord (At least in my experience) don't even know you can use them in DMs, and it's very often you end up using the guild from the interaction, and then if you forget to disable the command in DMs, that command won't work as there's no guild to be getting from the interaction.

**Comment by almostSouji** [comment, marked as answer]
Discord.js follows API spec. As documented here https://discord.com/developers/docs/interactions/application-commands#create-global-application-command you will see that being visible in DMs is the default for global application commands.

**Reply by sdanialraza** [reply]
Ah, that makes sense, so it's set up like that by Discord's API. Thank you for the answer!
